.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: true
+contact_links:
+  - name: Bitcoin Core Security Policy
+    url: https://github.com/bitcoin/bitcoin/blob/master/SECURITY.md
+    about: View security policy
```

Comment:
There's already a big green button for the security policy, so I don't think we need to add a second one manually.

Reply:
This config option enables the `Bitcoin Core Security Policy` row with the `Open` button which links to our `SECURITY.md`. The `Report a security vulnerability` row, with the big green `Report a vulnerability` button seems to be a GitHub default (which I don't know how to disable).

My preference would be to keep both, as otherwise the link to `SECURITY.md` via the `Report a vulnerability` button is from a small right menu item:

![](https://i.imgur.com/ICDRjEb.png)

Reply:
It looks like it's different depending on your repo permissions.

For me, the big green button says "View Policy" and clicking on it goes to https://github.com/willcl-ark/bitcoin/security/policy. For a repo I own/admin/maintain, it shows the page similar to the one in your screenshot.

Reply:
Hmmm, for me as a non admin this repo now looks like:

![image](https://user-images.githubusercontent.com/6606587/224306673-7d485dd4-d0d2-4410-8a4e-4f2e3f428a2b.png)

The green `View Policy` button takes me to a page which includes "Policy" and "Advisories", defaulting to showing the security policy `SECURITY.md`.

The row below that, "Bitcoin Core Security Policy" with the non-green `Open` button links me straight to `SECURITY.md`, which I agree is in theory somewhat redundant. If there is a preference to remove the second row, "Bitcoin Core Security Policy" direct link, then we could do that?